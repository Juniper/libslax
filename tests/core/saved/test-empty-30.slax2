version 1.2;

ns func = "http://exslt.org/functions";
ns test-ns = "http://xml.juniper.net/test";

main <test> {
    expr test-ns:test-func(0);
}

function test-ns:test-func ($x) {
    result "Between " _ $x;
}
