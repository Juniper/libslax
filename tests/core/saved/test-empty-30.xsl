<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:func="http://exslt.org/functions" xmlns:test-ns="http://xml.juniper.net/test" xmlns:slax-func="http://exslt.org/functions" version="1.0" extension-element-prefixes="slax-func">
  <xsl:template match="/">
    <test>
      <xsl:value-of select="test-ns:test-func(0)"/>
    </test>
  </xsl:template>
  <slax-func:function xmlns:slax-func="http://exslt.org/functions" name="test-ns:test-func">
    <xsl:param name="x"/>
    <slax-func:result xmlns:slax-func="http://exslt.org/functions" select="concat(&quot;Between &quot;, $x)"/>
  </slax-func:function>
</xsl:stylesheet>
