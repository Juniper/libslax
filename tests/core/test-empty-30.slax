version 1.1;

ns func = "http://exslt.org/functions";
ns test-ns = "http://xml.juniper.net/test";

match / {
    <test> {
	expr test-ns:test-func(0);
    }
}

function test-ns:test-func ($x) {
    result "Between " _ $x;
}
