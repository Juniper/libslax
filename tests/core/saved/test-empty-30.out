<?xml version="1.0"?>
<test xmlns:func="http://exslt.org/functions" xmlns:test-ns="http://xml.juniper.net/test">Between 0</test>
